src/features/rewards/profile/index.tsx

```diff
@@ -79,6 +81,13 @@ export default class Profile extends React.PureComponent<Props, {}> {
             </StyledProviderWrap>
             : null
           }
+          {
```

Comment:
I think this is not needed, because we already have prop `verified` that handles this use case. Please use that